src/main/java/org/broadinstitute/hellbender/utils/recalibration/BaseRecalibrationEngine.java

```diff
@@ -114,7 +115,7 @@ public void processRead( final GATKRead originalRead, final ReferenceDataSource
         final byte[] baqArray = nErrors == 0 ? flatBAQArray(read) : calculateBAQArray(read, refDS);
 
         if( baqArray != null ) { // some reads just can't be BAQ'ed
-            final ReadCovariates covariates = RecalUtils.computeCovariates(read, readsHeader, this.covariates);
+            final ReadCovariates covariates = RecalUtils.computeCovariates(read, readsHeader, this.covariates, true);
```

Comment:
Why do we still want to calculate the indel covariates here?  Is this something you plan on setting to false in a future PR?